script/RumpelConfig.sol

```diff
@@ -638,6 +670,244 @@ library RumpelConfig {
+            transferAllowState: RumpelGuard.AllowListState.ON,
+            approveAllowState: RumpelGuard.AllowListState.OFF
+        });
+        configs[5] = TokenGuardConfig({
+            token: MAINNET_MELLOW_HYVEX,
+            transferAllowState: RumpelGuard.AllowListState.ON,
+            approveAllowState: RumpelGuard.AllowListState.OFF
+        });
+        configs[6] = TokenGuardConfig({
+            token: MAINNET_MELLOW_RE7RTBTC,
+            transferAllowState: RumpelGuard.AllowListState.ON,
+            approveAllowState: RumpelGuard.AllowListState.OFF
+        });
+        configs[7] = TokenGuardConfig({
+            token: MAINNET_MELLOW_IFSETH,
+            transferAllowState: RumpelGuard.AllowListState.ON,
+            approveAllowState: RumpelGuard.AllowListState.OFF
+        });
+        configs[8] = TokenGuardConfig({
+            token: MAINNET_MELLOW_CP0XLRT,
+            transferAllowState: RumpelGuard.AllowListState.ON,
+            approveAllowState: RumpelGuard.AllowListState.OFF
+        });
+        configs[9] = TokenGuardConfig({
+            token: MAINNET_MELLOW_URLRT,
+            transferAllowState: RumpelGuard.AllowListState.ON,
+            approveAllowState: RumpelGuard.AllowListState.OFF
+        });
+        configs[10] = TokenGuardConfig({
+            token: MAINNET_MELLOW_COETH,
+            transferAllowState: RumpelGuard.AllowListState.ON,
+            approveAllowState: RumpelGuard.AllowListState.OFF
+        });
+        configs[11] = TokenGuardConfig({
+            token: MAINNET_MELLOW_HCETH,
+            transferAllowState: RumpelGuard.AllowListState.ON,
+            approveAllowState: RumpelGuard.AllowListState.OFF
+        });
+        configs[12] = TokenGuardConfig({
+            token: MAINNET_MELLOW_ISETH,
+            transferAllowState: RumpelGuard.AllowListState.ON,
+            approveAllowState: RumpelGuard.AllowListState.OFF
+        });
+        configs[13] = TokenGuardConfig({
+            token: MAINNET_MELLOW_SIBTC,
+            transferAllowState: RumpelGuard.AllowListState.ON,
+            approveAllowState: RumpelGuard.AllowListState.OFF
+        });
+        configs[14] = TokenGuardConfig({
+            token: MAINNET_MELLOW_LUGAETH,
+            transferAllowState: RumpelGuard.AllowListState.ON,
+            approveAllowState: RumpelGuard.AllowListState.OFF
+        });
+        configs[15] = TokenGuardConfig({
+            token: MAINNET_MELLOW_ROETH,
+            transferAllowState: RumpelGuard.AllowListState.ON,
+            approveAllowState: RumpelGuard.AllowListState.OFF
+        });
+        configs[16] = TokenGuardConfig({
+            token: MAINNET_MELLOW_RSUNIBTC,
+            transferAllowState: RumpelGuard.AllowListState.ON,
+            approveAllowState: RumpelGuard.AllowListState.OFF
+        });
+        configs[17] = TokenGuardConfig({
+            token: TBTC,
+            transferAllowState: RumpelGuard.AllowListState.ON,
```

Comment:
Should this be instant `PERMANENTLY_ON` ?

Reply:
Discussed in discord, but to mention again here – will do the `PERMANENTLY_ON` together with the blocklist